src/canopy/llm/octoai.py

```diff
@@ -0,0 +1,55 @@
+from typing import Optional, Any
+import os
+from canopy.llm import OpenAILLM
+from canopy.llm.models import Function
+from canopy.models.data_models import Messages
+
+OCTOAI_BASE_URL = "https://text.octoai.run/v1"
+
+
+class OctoAILLM(OpenAILLM):
```

Comment:
Can we add tests for OctoAILLM? Feel free to use the fixture in tests/system/llm/test_openai.py or create a new test_octoai.py file under tests/system/llm